.travis.yml

```diff
@@ -48,25 +52,23 @@ matrix:
   include:
     # Clang 3.5 - Release - Node
     - os: linux
-      sudo: false
+      sudo: required
+      dist: trusty
       language: node
-      compiler: ": node4-clang35-release"
+      compiler: "node4-clang35-release"
       env: BUILDTYPE=Release _CXX=clang++-3.5 _CC=clang-3.5
       addons: *clang35
       script:
         - nvm install 4
         - nvm use 4
         - make node
         - make test-node
-      after_script:
-        - ccache --show-stats
-        - ./platform/node/scripts/after_script.sh ${TRAVIS_JOB_NUMBER} ${TRAVIS_TAG:-}
```

Comment:
Why did you remove the `after_script`? This is used to upload test-suite results to S3, so we can review them.